features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2812,6 +2812,53 @@ CREATE TRIGGER IDN_OAUTH2_AUTHZ_CODE_ACTOR_TRIG NO CASCADE
         END
 /
 
+CREATE TABLE VC_CONFIG (
```

Comment:
Shall we have it as `IDN_VC` or `IDN_VC_TEMPLATE`, I think IDN prefix is added for all the identity claims

Reply:
fixed